CMakeLists.txt

```diff
@@ -337,7 +337,7 @@ abstract_language_plugin(en@dv englishdvorak
     FILES src/overrides.csv
     LIBRARIES westernsupport)
 abstract_language_plugin(fr-ch french LIBRARIES westernsupport)
-abstract_language_plugin(th thai)
+abstract_language_plugin(th thai LIBRARIES westernsupport DIRECTORY qml/keys)
```

Comment:
Huh? Thai language plugin should not be using `westernsupport`. I remembered not using it when I wrote it.

Reply:
Without it, the plugin fails to load with `undefined symbol: _ZN22AbstractLanguagePlugin16staticMetaObjectE` causing the keyboard process to crash.

It seems the AbstractLanguagePlugin was moved to westernsupport long ago when converted to CMake. I guess that should be fixed separately though, and perhaps affects other plugins too.